models/migrations/v109.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"xorm.io/xorm"
+)
+
+// BranchProtectionAddEnableWhitelist migrates can push vs whitelist push and enabling of approvals whitelist
+func BranchProtectionAddEnableWhitelist(x *xorm.Engine) error {
```

Comment:
```suggestion
func addBranchProtectionAddEnableWhitelist(x *xorm.Engine) error {
```

Reply:
or some other better name .. but it should start with add

Reply:
I tried to give it a slightly better name.